config/tips.txt

```diff
@@ -0,0 +1,136 @@
+Maintenance drones are deactivatable using your ID Card.
+Sometimes when the odds are stacked against you it's better to surrender than die.
+Toilet cisterns can store a plethora of different items if crowbarred open, just make sure to close them afterwards.
+Toilet mirrors can be opened using Alt + Click, you can store items discreetly inside.
+Disposal outlets make for great hidden storage provided you turn the pumps off.
+Riding down disposal chutes is a great way to end up in the Mourge.
+Enabling the neural lace option will allow you to be cloned, instead of being transferred to a synthetic body or borged.
+Pirate vessels come in sizes and armament. It's good to consider how well-armed you are for a fight.
+Pirate vessels usually contain a vault full of materials and loot, it's always worthwhile boarding them.
+Critically low blood-levels will cause brain damage. Iron + Sugar pills, IV drips with blood and eating food will help you restore blood levels.
+Lasers will cause burn damage and remove blood in your body.
+Exposure to low air environments will cause your lungs to collapse, always wear internals in low air environments to combat this.
+Not all exo-planets will have a breathable atmosphere, or safe temperatures. Shuttles will often have a sensor console that will have data to reflect the atmosphere.
+Some exo-planets may have flammable atmospheres, laser weaponry, smoking and gun-fire may set areas, or the entire planet on fire.
+Long-Range Holocommunicators allow you to talk to other vessels in the sector, provided someone is around on the other end to reply.
+Short-wave radios allow you to communicate with each-other in areas without telecommunications.
+Abandoned and derelict ruins may hold ancient treasures, or horrors.
+No two alien anomalies will have the same activation state, use the anomaly computer to analyze and outline what you may need to do to activate them.
+It is considered poor taste to decapitate someone and keep their neural lace, always give people a chance to come back into the round in some way.
+Straightjackets and Electropacks should never be used without admin approval, as they are impossible to get out of.
+Stowaways have a chance to be spawnable after the round starts, however as a Stowaway you won't be defined on the crew manifest or have any ID.
+Most ship AI will start with the manifest lawset, making it follow orders of crew on the ship manifest, as defined by the Captain.
+Ionispheric anomalies may cause AI's to gain new laws, making them act in unusual or fun manners.
+You can use Intellicards to card AI's into a portable state, allowing them to be repaired and have their lawsets visible.
+Rule Lawyering as an AI or Borg with lawsets or custom lawsets defined by antagonists is not fun for anyone.
+Rule Lawering as a imprint implanted person is not fun for anyone, especially the antagonist who used it on you.
```

Comment:
"Rule lawyering" is a bit of an unusual term that not many (especially new players) might understand. Could we explain this a little better? Also typo :P
```suggestion
Rule lawyering as an AI or Borg with lawsets or custom lawsets defined by antagonists is not fun for anyone.
Rule lawyering as an imprint-implanted person is not fun for anyone, especially the antagonist who used it on you.
```